MySQL8: cannot save notebook item

### Description

Can't save a notebook item when using MySQL8.

### Steps to Reproduce

1. Try to save a new notebook item as a student

**Expected behavior:**
A notebook item should be created

**Actual behavior:**
An error is thrown on the server-side and the notebook item is not created.
```
Caused by: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'groups, localNotebookItemId, nodeId, parentNotebookItemId, periodId, runId, serv' at line 1
```

### Additional Information
This is because 'groups' is a keyword starting in MySQL8.0.2.

--- a/src/main/java/org/wise/vle/domain/work/NotebookItem.java
+++ b/src/main/java/org/wise/vle/domain/work/NotebookItem.java
@@ -88,7 +88,7 @@ public class NotebookItem extends PersistableDomain {
   @Column(name = "parentNotebookItemId")
   private Integer parentNotebookItemId;
 
-  @Column(name = "groups")
+  @Column(name = "`groups`")
   private String groups;
 
   @Column(name = "type", length = 30, nullable = true)

